chainer/datasets/tuple_dataset.py

```diff
@@ -0,0 +1,37 @@
+import six
+
+
+class TupleDataset(object):
+
+    """Dataset of a tuple of datasets.
+
+    It combines multiple datasets into one dataset. Each example is represented
+    by a tuple whose i-th item corresponds to the i-th dataset.
```

Comment:
I commented like this because the mathematical symbol `i` is enclosed [here](https://github.com/pfnet/chainer/pull/1285/files#diff-2c3d279465c4c6c06840bb8a88ce6d9cR14). Do you have any reason for the difference?